htmlproof.bash

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+function err() {
+    msg "$*" 1>&2
```

Comment:
You're going to need the `msg` function too.

Reply:
good idea! I'm fine with putting these at the top of bash scripts.